package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] You need a way like `npm run` or `npm start` in order to start your app, see an example below:
![image](https://user-images.githubusercontent.com/62705947/208693283-d03033f1-cbf3-446d-9613-9e119076d09f.png)

Reply:
webpack-dev-server is configured